tests/pipelines/pixart_sigma/test_pixart.py

```diff
@@ -336,7 +336,7 @@ def test_pixart_1024(self):
         image = pipe(prompt, generator=generator, num_inference_steps=2, output_type="np").images
 
         image_slice = image[0, -3:, -3:, -1]
-        expected_slice = np.array([0.0742, 0.0835, 0.2114, 0.0295, 0.0784, 0.2361, 0.1738, 0.2251, 0.3589])
```

Comment:
These values are from a different device I think. On main this test fails as well without the updated slice.